### ISSUE TITLE
Error with Redmine 2.3.2

### ISSUE THREAD

**Issue by vietstone-ng** [zeyuLiang/redmine_customized_report#9]
Hi, 
It's run OK with Redmine 2.5.2, but not OK with 2.3.2. There's many errors of undefined method or attribute. Please help me to run with Redmine 2.3.2
Thank you.

**Comment by zeyuLiang** [comment]
sorry, but i don't intent to keep it working for older redmine versions~so you should update your redmine to 2.5.0+

**Comment by vietstone-ng** [comment]
I want to create a version for 2.3.2. Do you recommend?

**Comment by zeyuLiang** [comment]
feel free to do it~^_^ .May not be too hard~

**Comment by vietstone-ng** [comment]
Yes. Do you recommend any specific technique or mechanism by Redmine? I have read through the code, class Issue, Query and IssueQuery by Redmine, class ReportQuery by your plugin, but I can not get the idea. Thank you.

**Comment by vietstone-ng** [comment]
Could you tell me the role of 2 classes CustomField and IssueCustomField? Thank you
